client/src/components/DataFiles/DataFilesListing/DataFilesListing.jsx

```diff
@@ -153,12 +153,18 @@ const DataFilesListing = ({ api, scheme, system, path, isPublic }) => {
   const systemDisplayName = useSystemDisplayName({ scheme, system, path });
   const homeDir = selectedSystem?.homeDir;
 
-  // Check if the current path is the home directory itself
-  const isAtHomeDir = path.replace(/^\/+/, '') === homeDir?.replace(/^\/+/, '');
+  // Check if the current path is the root directory
+  const isRootDir = path === '/' || path === '';
 
-  // Determine the sectionName based on the path (if homeDir, use systemDisplayName--else use current dir)
+  // Adjusted check for home directory
+  const isAtHomeDir =
+    isRootDir || path.replace(/^\/+/, '') === homeDir?.replace(/^\/+/, '');
+
+  // Determine the sectionName with added handling for root directory
   const sectionName = isAtHomeDir
```

Comment:
nit: for code readability, is it possible to not use nested ternary operator?

Reply:
@chandra-tacc Yes and no. When in the root directory (or deeper folders until you get to the home dir) the dropdown will display the systemName for that item. It's like that on prod too for Frontera and Public Data, so technically it's not due to this PR, but the original dropdown PR. Also, clicking on 'Corral' for example will take you back to the root dir of that system. So it's a label issue based off a path that isn't the root or home dir, but I can see how the user would be confused by this when navigating in root.

Reply:
@tjgrafft  - thanks for sharing those details. I agree it is not related to this. It would be good to log a bug and fix it separately.

Reply:
How about using a function instead?
```
function determineSectionName(isAtHomeDir, isRootDir, systemDisplayName, path) {
  if (isAtHomeDir) {
    return isRootDir ? 'Root' : systemDisplayName;
  } 
  return getCurrentDirectory(path);
}

const sectionName = determineSectionName(isAtHomeDir, isRootDir, systemDisplayName, path);
```

Reply:
@chandra-tacc sounds good on logging a bug and fixing it separately